rules/50-udev-default.rules.in

```diff
@@ -32,6 +32,7 @@ SUBSYSTEM=="input", KERNEL=="js[0-9]*", MODE="0664"
 SUBSYSTEM=="video4linux", GROUP="video"
 SUBSYSTEM=="graphics", GROUP="video"
 SUBSYSTEM=="drm", GROUP="video"
+SUBSYSTEM=="drm", KERNEL=="renderD*", GROUP="render", MODE="@GROUP_RENDER_MODE@"
```

Comment:
hmm, i figure the line before this one you are adding should be changed to `SUBSYSTEM="drm", KERNEL!="renderD*", …` because otherwise we'll first assign these devices "video", and then override this with "render" in the next step, which is slightly ugly

Reply:
also, it might make sense to reorder this a bit: place the lines with GROUP="render" in its own block, right after the lines with GROUP="video" but separated by an empty newline. i.e. keep them close, but separate, as they are managed differently now.